.eslintrc.js

```diff
@@ -49,6 +49,21 @@ module.exports = {
         'import/no-extraneous-dependencies': 'off',
       },
     },
+    {
+      files: ['scripts/**/*'],
+      rules: {
+        'no-void': 'off',
+        'no-console': 'off',
+        'no-process-exit': 'off',
+        'functional/no-expression-statement': 'off',
+        '@typescript-eslint/naming-convention': 'off',
+        'import/no-extraneous-dependencies': 'off',
+        'jsdoc/require-description': 'off',
+        'jsdoc/match-description': 'off',
+        'jsdoc/require-description-complete-sentence': 'off',
```

Comment:
Have you noticed `eslint-plugin-jsdoc` is really buggy in VSCode and stales the process sometimes?

Reply:
Does it? I never saw this happen. Shall we fix it separately?

Reply:
I don't use it.